.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{

```

Comment:
Do you intend to add this?  Like does this make life easier for vscode users in some way?  I'm not necessarily adverse to it but it seems like the sort of thing that would normally be purely local to individual users.

Reply:
No, of course. I don't know where that came from... obviously it was added by GitHub tooling when I made fixes